fiat-roles/src/main/java/com/netflix/spinnaker/fiat/permissions/PermissionsRepository.java

```diff
@@ -54,21 +56,21 @@
    */
   Optional<UserPermission> get(String id);
 
-  /** Gets all UserPermissions in the repository keyed by user ID. */
-  Map<String, UserPermission> getAllById();
+  /** Gets all Roles in the repository keyed by user ID. */
+  Map<String, Set<Role>> getAllById();
 
   /**
-   * Gets all UserPermissions in the repository that has at least 1 of the specified roles, keyed by
-   * user ID. Because this method is usually used in conjuction with updating/syncing the users in
-   * question, the returned map will also contain the unrestricted user. If anyRoles is null,
-   * returns the same result as getAllById() (which includes the unrestricted user). If anyRoles is
-   * empty, this is an indication to sync only the anonymous/unrestricted user. When this is the
-   * case, this method returns a map with a single entry for the unrestricted user.
+   * Gets all Roles in the repository that has at least 1 of the specified roles, keyed by user ID.
```

Comment:
As someone largely unfamiliar with the inner workings of Fiat persistence this doesn't really make sense? Why does a `Role` object have one of the specified roles - is a Role not just a single role?

Reply:
Yeah, that could do with a rewording. It's basically 

> Finds all users with at least one of the set of specified roles and returns a map of users all their associated roles.

I'll fix that up.